tensorflow/core/user_ops/gather_columns_functor.h

```diff
@@ -0,0 +1,125 @@
+#ifndef TENSORFLOW_USEROPS_GATHER_COLUMNS_FUNCTOR_H_
+#define TENSORFLOW_USEROPS_GATHER_COLUMNS_FUNCTOR_H_
+
+//#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+
+#include "tensorflow/core/framework/tensor_types.h"
+#include "../kernels/bounds_check.h" //--TODO: Currently <#include "tensorflow/core/kernels/bounds_check.h"> gives an error. Need to check and fix.
```

Comment:
The reason for this is because `$TF_INC = ~/.virtualenvs/py3/lib/python3.5/site-packages/tensorflow/include` does not include "tensorflow/core/kernels/bounds_check.h" within it. 

I have added an environment variable in my .bashrc (TF_ROOT=/home/nash/Dropbox/KTH/Projects/tensorflow) pointing to my TF root directory, and include the variable while compiling.